content/guides/intro-to-defi.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Intro to DeFi – How to navigate the Degen world'
+description: 'DeFi is one of the key areas in web3: the guide introduces the concept and provides resources for using and developing DeFi applications.'
+authors: ['@bitblondy', '@uttam_singhk']
+level: Beginner
+tags: ['Web3', 'DeFi']
+languages: ['Solidity']
+url: 'https://www.useweb3.xyz/guides/defi-intro'
+date: 2024-02-15
+dateAdded: 2024-02-17
+---
+
+## What is DeFi?
+
+Decentralised Finance (DeFi) is a term for web3 applications in the financial sector. They replicate the services of traditional financial service providers on a decentralised infrastructure, allowing permissionless access.
+
+They thus promise access to financial services without the need for a bank account or verification by a central provider, leading to greater financial inclusion. On the other hand, most of the volume is currently created by experienced DeFi users. Some individuals are even creating scam DApps, so be careful.
+
+DeFi is one of the main use cases of web3. Especially during the so-called "DeFi Summer" in 2020, there was an explosion in the use and popularity of DeFi applications. Most of the DeFi market is on Ethereum, but many applications are now on multiple chains. These typically allow for lower transaction costs.
+
+If you want to contribute to the guide, [have a look on Github](https://github.com/wslyvh/useWeb3/tree/main).
+
+### Some of the characteristics of DeFi:
+
+- **Permissionless**: Use your wallet instead of a bank account. Thanks to the peer-to-peer structure, there is no centralized entity that controls decision-making. However, your funds are your own responsibility.
+- **Global**: It doesn’t matter where you are, DeFi is a global phenomenon.
+- **Fast**: Transactions are finished rapidly compared to bank transfers, especially for cross-border transfers.
+- **Pseudonymous**: People act through their wallet addresses, no personal information needed.
+- **Transparent**: Transactions are public, though, and often the code of the applications is open-source.
+
+DeFi essentially extends the "digital money" premise of bitcoin beyond simply making payments with digital currency. DeFi applications allow you to trade crypto, create derivatives, borrow and lend, earn interest or buy insurance. In the next section, we will explain the broader areas of DeFi.
+
+Some intro pages and learning resources to start your DeFi journey:
+
+- [_Ethereum.org DeFi intro page_](https://ethereum.org/en/defi)
+- [_Rekt news_](https://rekt.news/de/)
+- [_Finematics Youtube channel_](https://www.useweb3.xyz/videos/finematics)
+- [_The Defiant news_](https://thedefiant.io/)
+- [_How to DeFi Advanced Book_](https://www.useweb3.xyz/books/how-to-defi-advanced)
```

Comment:
I wonder how up to date this still is?

Reply:
reight, from 2021, probably not the most up to date...